src/languages/en-US.ts

```diff
@@ -1381,6 +1381,15 @@ export default class extends Language {
 					a temporary ban.`,
 			examples: ['@Pete', '@Pete Spamming all channels.', '@Pete Spamming all channels, for 24 hours.']
 		}),
+		COMMAND_DEHOIST_DESCRIPTION: 'Shoot everyone with the dehoist-inator 3000',
+		COMMAND_DEHOIST_EXTENDED: builder.display('dehoist', {
+			extendedHelp: `The act of hoisting involves adding a character with a value of 64 and below in front of your nickname
```

Comment:
How bold of you to assume everyone knows what `64` is.
And much less what it represents in ASCII and why is it relevant.